.github/workflows/docs.yml

```diff
@@ -4,13 +4,19 @@ on:
   push:
     branches:
       - main
+  merge_group:
+  pull_request:
+    paths:
+      - "docs/**"
+      - "package.json"
+      - "pnpm-lock.yaml"
```

Comment:
The docs check isn't running on this pull request because none of these `paths` changed, but we can modify this so that it runs on any `pull_request`.